lib/aws/rails/middleware/ebs_sqs_active_job_middleware.rb

```diff
@@ -96,23 +96,13 @@ def in_docker_container_with_cgroup2?
         File.exist?('/proc/self/mountinfo') && File.read('/proc/self/mountinfo') =~ %r{/docker/containers/}
       end
 
-      def default_gw_ips
-        default_gw_ips = ['172.17.0.1']
```

Comment:
This just seems to resolve in giving me two entries of `172.17.0.1` on the AL2023 test, which seems legit. Maybe the issue is the non-remote ip/addr usage? IIRC when I ran this, the debug logger had the IP as something else. I can add more debug statements to prove what's going on (this is very weird).

Reply:
```
D, [2024-11-05T17:13:44.713584 #16] DEBUG -- : aws-sdk-rails middleware detected call from Elastic Beanstalk SQS Daemon.
D, [2024-11-05T17:13:44.713881 #16] DEBUG -- : SQSD request detected from IP: 127.0.0.1.
D, [2024-11-05T17:13:44.713926 #16] DEBUG -- : SQSD request detected from Remote IP: 127.0.0.1.
```

Maybe it needs to be remote_addr 🤯 . Trying again

Reply:
if this is the root cause—checking `request.ip` does not work—then I wonder if using the OG code is fine, it just needs to check against r**emote_ip and remote_addr** instead. I assume it would since it seemed like it was calculating properly. 

I'll make those changes once I confirm it really must be remote_addr

Reply:
`D, [2024-11-05T17:41:38.678283 #16] DEBUG -- : SQSD request detected from Remote ADDR: 172.17.0.1.`

HA! There it is.

Reply:
Updated code!